src/yargs-commands/schema/push.mjs

```diff
@@ -0,0 +1,122 @@
+import { container } from '../../cli.mjs'
+import { confirm } from "@inquirer/prompts";
+import { commonQueryOptions } from '../../lib/command-helpers.mjs';
+
+async function doPush(argv) {
+  const logger = (await container.resolve("logger"))
+  const fetch = container.resolve("fetch")
+
+  const gatherFSL = container.resolve("gatherFSL")
+  try {
+    const fsl = await gatherFSL(argv.dir)
+    if (argv.force) {
+      const params = new URLSearchParams()
+      if (argv.force)
+        params.set("force", "true")
+      if (argv.staged)
+        params.set("staged", "true")
+
+      const path = new URL(`/schema/1/update?${params}`, argv.url);
+      const res = await fetch(path.toString(), {
+        method: "POST",
+        headers: { AUTHORIZATION: `Bearer ${argv.secret}` },
+        body: fsl,
+        // https://github.com/nodejs/node/issues/46221
+        // https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/1483
+        // @ts-expect-error-next-line
```

Comment:
nit: is this compiled with typescript? or is it just javascript?

Reply:
raw js - the typescript annotations don't do anything, but this was copied from `src/commands/schema/push.ts`, so a few of the non-breaking TS flavored bits stuck around. i'll take these out now while i'm thinking about them.